### PR TITLE
Fix/desc dialog

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
This PR will show the upload dialog only once.

### CODE REVIEW

File: lib/repository/repository.dart [hunk 1]
```diff
@@ -80,18 +80,23 @@ abstract class Repository {
...
+  /// Output: [int] returns id of the inserted document & will return error in the form of [Failure]
   Future<Either<Failure, int>> saveNft(NFT nft);
 
   /// This method will update draft in the local database from description Page
   /// Input:[SaveNft] this model data contains bring [id],[nftName],[nftDescription],[creatorName],[step]
   /// Output: [bool] returns whether the operation is successful or not
   Future<Either<Failure, bool>> updateNftFromDescription({required SaveNft saveNft});
 
+  /// This method will update the draft of the NFT
+  /// Input: [id] of the draft that will be updating
```
Comment (mustafapylons) [inline lib/repository/repository.dart]: "updated" instead of "updating"

File: lib/repository/repository.dart [hunk 2]
```diff
@@ -80,18 +80,23 @@ abstract class Repository {
...
+  /// Output: [bool] returns that draft is update or not & will return error in the form of [Failure]
```
Comment (mustafapylons) [inline lib/repository/repository.dart]: "updated" instead of "update"

**Review by mustafapylons — COMMENTED** [review]
some typos, otherwise code is good.

**Review by Amean — APPROVED** [review]

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Pull request merged**